### DISCUSSION TITLE
PC1616 Quick exit - how to send?

### DISCUSSION THREAD

**Discussion by danergo** [taligentx/dscKeybusInterface#129] [dscKeybusInterface Discussion]
Hi,

I've just started to look into this nice library. Thank you!

I could not yet find out how can I send the quick exit code which the keypad have as button on the bottom-right corner:

![PK5508_2_](https://user-images.githubusercontent.com/11708344/71787308-63343100-3016-11ea-929c-643450f20109.jpg)

Do we have the keypad codes already decoded for it?

**Comment by Dilbert66** [comment]
With my DSC system you send code *0  as  dsc.write("*0")

**Comment by danergo** [comment]
Thanks, that was the combination I've searched for.
